src/devices/Board/RaspberryPiBoard.cs

```diff
@@ -520,6 +615,215 @@ public override PinUsage DetermineCurrentPinUsage(int pinNumber)
             return PinUsage.Unknown;
         }
 
+        /// <summary>
+        /// Checks if the I2C overlay is activated in the configuraztion file.
+        /// </summary>
+        /// <returns>True if it is.</returns>
+        public bool IsI2cOverlayActivate()
```

Comment:
nit: IsI2cOverlayActive or just IsI2cActived

Reply:
I'm open for any, I'll adjust then SPI and PWM. Any preference?

Reply:
I simplified, makes more sense